assets/device/classes/thermostat.json

```
@@ -18,7 +20,9 @@
     "sv": "Använd den här enhetsklassen för termostater, antingen för hela hemmet eller elementmonterade, oftast tillsammans med funktionerna `mät_temperatur`, `måltemperatur` och `termostatläge`.",
     "no": "Bruk denne enhetsklassen for termostater, enten for hele hjemmet eller monterte på varmeelement, ofte sammen med funksjonene `måling_temperatur`, `ønsket_temperatur` og `termostat_modus`.",
     "es": "Utiliza esta clase de dispositivo para los termostatos, tanto los de toda la casa como los montados en los radiadores, generalmente junto con las capacidades 'medir_temperatura', 'temperatura_configurada' y 'modo_termostato'.",
-    "da": "Brug denne enhedsklasse til termostater, enten til hele hjemmet eller på radiatorer, normalt sammen med `mål_temperatur`, `måltemperatur` og `termostattilstand` funktionerne."
+    "da": "Brug denne enhedsklasse til termostater, enten til hele hjemmet eller på radiatorer, normalt sammen med `mål_temperatur`, `måltemperatur` og `termostattilstand` funktionerne.",
```

```suggestion
    "da": "Brug denne enhedsklasse til termostater, enten til hele hjemmet eller på radiatorer, normalt sammen med `measure_temperature`, `target_temperature` og `thermostat_mode` funktionerne.",
```
This should also be fixed for other translations in this description